### DISCUSSION TITLE
Query with join table ?

### DISCUSSION THREAD

**Discussion by tphuc** [supabase/supabase#3552] [Questions] [answered]
I have 3 tables below with foreign keys set up.
| products |         
|----------|    
| id             | 

|categories|     
|----------|
|id              |

|product_category|
|------------------|
|product_id|
|category_id|

I want to query all `products` match with  a specific `category`. This is my attempt but it doesn't work.
```js
await supabase.from('products')
.select(`
     *,
     categories(
        *
     )
`)
.filter('product_category.category_id', 'eq', ANY_CATEGORY_ID)
```
The query above returns **all** `products`, not what I want. Any helps is appreciated, thank you.

**Comment by steve-chavez** [comment, marked as answer]
Check this issue: https://github.com/supabase/postgrest-js/issues/197. This feature will come soon.

For now you can use a view or function.

(Related to https://github.com/supabase/supabase/discussions/3546)

**Reply by tphuc** [reply]
Do you know what is the release date (or estimation date) of this feature ?

**Comment by MadCoderme** [comment]
Hi! Is there any update on this feature?
